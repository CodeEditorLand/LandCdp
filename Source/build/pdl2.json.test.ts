/*---------------------------------------------------------
 * Copyright (C) Microsoft Corporation. All rights reserved.
 *--------------------------------------------------------*/

import { pdlStringToJson } from './pdl2json';

describe('pdl2json', () => {
	it('is sane', async () => {
		const obj = await pdlStringToJson(`
version
  major 1
  minor 3

domain Foo
  type Bar extends object
    properties
      integer baz
`);
		expect(obj).toMatchSnapshot();
	});
});
