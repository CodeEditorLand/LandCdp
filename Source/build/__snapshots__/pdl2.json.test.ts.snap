// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pdl2json is sane 1`] = `
Object {
  "domains": Array [
    Object {
      "domain": "Foo",
      "types": Array [
        Object {
          "id": "Bar",
          "properties": Array [
            Object {
              "name": "baz",
              "type": "integer",
            },
          ],
          "type": "object",
        },
      ],
    },
  ],
  "version": Object {
    "major": "1",
    "minor": "3",
  },
}
`;
